fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -359,7 +362,8 @@ struct CommitTransactionRef {
 	}
 
 	size_t expectedSize() const {
-		return read_conflict_ranges.expectedSize() + write_conflict_ranges.expectedSize() + mutations.expectedSize();
+		return read_conflict_ranges.expectedSize() + write_conflict_ranges.expectedSize() + mutations.expectedSize() +
+		       tenantIds.expectedSize();
```

Comment:
I don't know that we want to add this to `expectedSize`. In practice it probably doesn't make much different, but before the number more closely correlated with the size of the commit as we normally think of it (mutations + conflict ranges).

Reply:
Yeah. It seems like we even don't need to serialize it.